src/p10/p10_edbgEcmdDllScom.C

```diff
@@ -220,7 +220,7 @@ uint32_t p10_dllGetScom(const ecmdChipTarget & i_target, uint64_t i_address, ecm
                   continue;
           }
 
-          const char *path = pdbg_target_path(target);
+          const char *pPath = pdbg_target_path(target);
```

Comment:
existing path variable is unused so we can remove it and also do not use pPath rather use path itself.

Reply:
existing path variable is used inside the else, so limited the path variable scope to where it is used. So i had to create the new variable as it falls within the same scope. I can limit the original path scope by adding empty braces. But I believe that would not be accepted. right?